.targets.mk

```diff
@@ -1,4 +1,9 @@
-TARGETS_DRAFTS := draft-gpew-priv-ppm
-TARGETS_TAGS := 
-draft-gpew-priv-ppm-00.md: draft-gpew-priv-ppm.md
-	sed -e 's/draft-gpew-priv-ppm-latest/draft-gpew-priv-ppm-00/g' $< >$@
+TARGETS_DRAFTS := draft-gpew-priv-ppm 
```

Comment:
You shouldn't be checking this file in.  Especially as it is listed in .gitignore.

Reply:
Ugh, thanks for flagging! Reverted.

However, after `make clean && make` I see the file in `git status`, despite the fact that it's listed as ignored:
```
$ git status
On branch cjpatton/replay
Your branch is up to date with 'origin/cjpatton/replay'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   .targets.mk

no changes added to commit (use "git add" and/or "git commit -a")
```